### PR TITLE
[GH-15608] Replace Jira links with GH issue equivalent [nocheck]

### PR DESCRIPTION
**Pull request by mn-mikke**
closes: #15608

### CODE REVIEW

**Review by wendycwong — CHANGES_REQUESTED** [review]

File: h2o-docs/src/product/automl.rst [hunk 1]
```diff
@@ -539,7 +539,7 @@ FAQ
 
   AutoML includes `XGBoost <data-science/xgboost.html>`__ GBMs (Gradient Boosting Machines) among its set of algorithms. This feature is currently provided with the following restrictions:
 
-  - XGBoost is not currently available on Windows machines, or on OS X with the new Apple M1 chip.  Please check the JIRA tickets for `Windows <https://h2oai.atlassian.net/browse/PUBDEV-8523>`__ and `Apple M1 <https://h2oai.atlassian.net/browse/PUBDEV-8482>`__ for updates. 
+  - XGBoost is not currently available on Windows machines, or on OS X with the new Apple M1 chip.  Please check the JIRA tickets for `Windows <https://github.com/h2oai/h2o-3/issues/7139>`__ and `Apple M1 <https://github.com/h2oai/h2o-3/issues/7180>`__ for updates.
```
[the remark this answers is not in the document]
Comment (mn-mikke) [reply h2o-docs/src/product/automl.rst]: changed

File: h2o-docs/src/product/data-science/xgboost.rst [hunk 2]
```diff
@@ -308,7 +308,7 @@ This section provides a list of XGBoost limitations - some of which will be addr
 
 The list of limitations include:
 
-  - XGBoost is not currently supported on Windows or on the new Apple M1 chip.  Please check the JIRA tickets for `Windows <https://h2oai.atlassian.net/browse/PUBDEV-8523>`__ and `Apple M1 <https://h2oai.atlassian.net/browse/PUBDEV-8482>`__ for updates. 
+  - XGBoost is not currently supported on Windows or on the new Apple M1 chip.  Please check the JIRA tickets for `Windows <https://github.com/h2oai/h2o-3/issues/7139>`__ and `Apple M1 <https://github.com/h2oai/h2o-3/issues/7180>`__ for updates.
```
[the remark this answers is not in the document]
Comment (mn-mikke) [reply h2o-docs/src/product/data-science/xgboost.rst]: changed

File: h2o-docs/src/booklets/v1_2014/source/DeepLearningRVignette.tex [hunk 3]
```diff
@@ -32,11 +32,26 @@
...
+Retail and Payment Systems.
+
+Using in-memory compression techniques, H2O can handle billions of data rows in-memory — even with a fairly small cluster. The platform includes interfaces for R, Python, Scala, Java, JSON and Coffeescript/JavaScript, along with its built-in Flow web interface that make it easier for non-engineers to stitch together complete analytic workflows. The platform was built alongside (and on top of) both Hadoop and Spark Clusters and is typically deployed within minutes.
+
+H2O implements almost all common machine learning algorithms — such as generalized linear modeling (linear regression, logistic regression, etc.), Na\"{i}ve Bayes, principal components analysis, time series, k-means clustering and others. H2O also implements best-in-class algorithms such as Random Forest, Gradient Boosting and Deep Learning at scale. Customers can build thousands of models and compare them to get the best prediction results.
```
[the remark this answers is not in the document]
Comment (mn-mikke) [reply h2o-docs/src/booklets/v1_2014/source/DeepLearningRVignette.tex]: changed

File: h2o-docs/src/booklets/v1_2014/source/GLM_Vignette.tex [hunk 4]
```diff
@@ -40,11 +40,26 @@ \section{Introduction} \label{1}
...
+Retail and Payment Systems.
+
+Using in-memory compression techniques, H2O can handle billions of data rows in-memory — even with a fairly small cluster. The platform includes interfaces for R, Python, Scala, Java, JSON and Coffeescript/JavaScript, along with its built-in Flow web interface that make it easier for non-engineers to stitch together complete analytic workflows. The platform was built alongside (and on top of) both Hadoop and Spark Clusters and is typically deployed within minutes.
+
+H2O implements almost all common machine learning algorithms — such as generalized linear modeling (linear regression, logistic regression, etc.), Na\"{i}ve Bayes, principal components analysis, time series, k-means clustering and others. H2O also implements best-in-class algorithms such as Random Forest, Gradient Boosting and Deep Learning at scale. Customers can build thousands of models and compare them to get the best prediction results.
```
[the remark this answers is not in the document]
Comment (mn-mikke) [reply h2o-docs/src/booklets/v1_2014/source/GLM_Vignette.tex]: changed

**Comment by hannah-tillman** [comment]
I am going through the docs on your branch to find any other mentions of Jira, and I found in Flow we are [still pointing to Jira when you report an issue](https://docs.h2o.ai/h2o/latest-stable/h2o-docs/flow.html#reporting-issues). So this section needs to be updated, but so does the UI.

File: docker/jenkins-images/Dockerfile-graalvm [hunk 5]
```diff
@@ -1,4 +1,4 @@
-# This is not part of the automated system (Will be fixed by https://h2oai.atlassian.net/browse/ITA-1506)
```
Comment (valenad1) [inline docker/jenkins-images/Dockerfile-graalvm:1]: ITA are not converted?

File: scripts/jenkins/groovy/buildConfig.groovy [hunk 6]
```diff
@@ -323,6 +323,7 @@ class BuildConfig {
               change.lastIndexOf("pyunit_") > change.lastIndexOf("/") && // Allow to run "pyunit_*" files inside of "pyunit_*" directory but do not allow e.g. utilsPY.py to run
               !change.contains("pyunit_explain") && !change.endsWith("_large.py") && // takes too much time to run
               !change.contains("pyunit_h2oassembly_download_mojo")
+              !change.contains("test_cloud")
```
[the remark this answers is not in the document]
Comment (mn-mikke) [reply scripts/jenkins/groovy/buildConfig.groovy:326]: Yes, this is an intended change, it fixes the problem in the modified tests. it includes all tests under `h2o-py`, but then it reports:
ERROR: Test does not exist: pyunit_read1GB_medium.py

http://mr-0xg1:8080/blue/organizations/jenkins/h2o-3-pipeline/detail/PR-15610/2/pipeline

File: docker/jenkins-images/Dockerfile-graalvm (hunk 5 above)
Comment (mn-mikke) [reply docker/jenkins-images/Dockerfile-graalvm:1]: No, they are unimportant - just technical or devops tasks.

**Comment by mn-mikke** [comment]
@hannah-tillman Good point! We will need to update flow ui as well.

**Pull request merged**